test/common-wrapper.js

```diff
@@ -60,16 +60,16 @@ module.exports = function(wrapper, defNumPorts) {
         it("should accept object with one inPort definition", function() {
             return Promise.resolve({
                 inPorts: { 
-                    myinput: { datatype: 'string', 
+                    input: { datatype: 'string', 
```

Comment:
Do we really have to hard code the input parameter name?  That feels a bit kludgey